cloudfront/iiif/edge-lambda/src/dlcs_path_rewrite.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as origin from './dlcs_path_rewrite';
+import testRequest from './test_event_request';
+import { Context } from 'aws-lambda';
+
+interface ExpectedRewrite {
+  in: string
+  out: string
+}
+
+const rewrite_tests = (): Array<ExpectedRewrite> => {
+  return [
+    {
+      in: '/image/B0009851.jpg/full/1338%2C/0/default.jpg',
+      out: '/B0009851/full/1338%2C/0/default.jpg'
+    },
+    {
+      in: '/image/B0009852.JPG/full/1338%2C/0/default.jpg',
+      out: '/B0009852/full/1338%2C/0/default.jpg'
+    },
+    {
+      in: '/image/B0009853/full/1338%2C/0/default.jpg',
+      out: '/B0009853/full/1338%2C/0/default.jpg'
+    }
+  ]
```

Comment:
Having not looked at the implementation yet, I'd suggest a couple of other test cases:

* What if the output format is something other than `.jpg`, e.g. `/default.png`
* What if the region isn't `full`? e.g. `B0009853/400,400,200,200`
* What if the quality isn't `default`? e.g. `/gray.jpg`

Reply:
Good point. Especially as i'm going to add to the things that get rewritten soon!